src/Extension/CoreExtension.php

```diff
@@ -1655,4 +1655,29 @@ function twig_check_arrow_in_sandbox(Environment $env, $arrow, $thing, $type)
         throw new RuntimeError(sprintf('The callable passed to the "%s" %s must be a Closure in sandbox mode.', $thing, $type));
     }
 }
+
+    /**
+     * Check if object attribute regardless of the value, it should return true
+     * It should be able to handle when object is ActiveRecord and have null value for a column e.g $user.RoleId
+     *
+     * @param $object
+     * @param $attribute
+     * @return bool
+     */
+    function hasAttribute($object, $attribute)
+    {
+        if (!is_object($object)) {
```

Comment:
This check can be removed as this is always an object at this point (see line 1455).